### ISSUE TITLE
ONNX Export

### ISSUE THREAD

**Issue by soyebn** [rwightman/efficientdet-pytorch#32]
Have you tried ONNX export for **tf_efficientdet_lite0**? I tried with exporting only backbone+BiFPN by defining forward_dummy similar to used in mmdetect. I got very complicated ONNX graph (Scatter/Gather/Div/Squeeze etc..) Whereas efficientNetLite models you exported using **gen-efficientnet-pytorch/blob/master/onnx_export.py** are very clean. 

Is there a way to get clean ONNX graph for tf_efficientdet_lite0? 

Thanks a lot for all your help.

**Comment by rwightman** [comment]
Someone else was trying for a commercial application and ran into issues, I think the model minus the bench might be somewhat reasonable to export, but doubt the post processing, nms, etc will work at all without doing some custom work. 

This is not something I plan to tackle here. It's something I'd look into under contract.

**Comment by jinfagang** [comment]
@rwightman  How many AP does it got in this repo with efficientdet D3?

**Comment by rwightman** [comment]
@jinfagang I'm not sure, I don't have the hardware resources to train a D3 in a reasonable timeframe or with a large enough batch size. It should get fairly close to the original if you have enough GPUs to get the effective batch size into the 64 range (and you're using sync-bn).

**Comment by jinfagang** [comment]
@rwightman With another git repo, D3 hard to on custom dataset.

**Comment by rwightman** [comment]
@jinfagang every custom dataset is different so hard to say, need to experiment with your hparams... I know a few people have trained D0 to D2 models for custom datasets that work well using this code, on Kaggle challenges, Waymo self driving challenge, etc ... better results than the other git repos (so I've been told)

**Comment by jinfagang** [comment]
@rwightman  thanks, will try this repo

**Comment by glenn-jocher** [comment]
@rwightman thanks for working through all the training for this. I know it's a grueling process. 

What training times are you seeing for D0 with your implementation? Do you have loss/mAP plots vs epoch you could show?

**Comment by rwightman** [comment]
@glenn-jocher I have the csv in my training history folders somewhere, I'll dig it up and attach it here when I get a moment
